packages/workflow/src/workflow.ts

```diff
@@ -1318,6 +1319,28 @@ export function setDefaultSignalHandler(handler: DefaultSignalHandler | undefine
   }
 }
 
+/**
+ * Set a query handler function that will handle query calls for non-registered query names.
+ *
+ * Queries are dispatched to the default query handler in the order that they were accepted by the server.
+ *
+ * If this function is called multiple times for a given query name the last handler will overwrite any previous calls.
+ *
+ * @param handler a function that will handle queries for non-registered query names, or `undefined` to unset the handler.
+ */
+export function setDefaultQueryHandler(handler: DefaultQueryHandler | undefined): void {
+  const activator = assertInWorkflowContext(
+    'Workflow.setDefaultQueryHandler(...) may only be used from a Workflow Execution.'
+  );
+  if (typeof handler === 'function') {
```

Comment:
nit: You can combine the `if` and `else if` branches here.

Reply:
Done.